packages/react-aria-components/docs/Table.mdx

```diff
@@ -1208,6 +1208,75 @@ function Example() {
 }
 ```
 
+## Dependencies
+
+The `dependencies` prop can be used to ensure that separate parts of the collection update in response to each other, for example, column additions and removals.
+Be careful using it too frequently as it will cause the table to re-render more often than necessary which can cause performance degradation.
+Use the same strategies for reducing unnecessary invalidations as you would with any other hook.
+
+This prop is an array of dependencies that will trigger a cache invalidation within the collection. It should follow the same rules as React various hooks with dependencies arrays.
+> The list of dependencies must have a constant number of items and be written inline like `[dep1, dep2, dep3]`.
```

Comment:
link to React docs?

Reply:
We pass the dependencies directly to useMemo internally :)